### PR TITLE
Pub 1843 Email subscriptions

### PR DESCRIPTION
**Pull request by ashwini-mv**
### JIRA link

https://tools.hmcts.net/jira/browse/PUB-1843

### Change description

Added functional tests for email subscriptions

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/test/end-to-end/tests/verified/email-subscriptions-test.ts [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+        I.waitForText('Bulk unsubscribe');
+        I.click('#bulk-unsubscribe-button');
+        I.waitForText('There is a problem');
+        I.see('At least one subscription must be selected');
+        I.click(locate('//tr').withText(locationName).find('input').withAttr({ id: 'courtSubscription' }));
+        I.click('#bulk-unsubscribe-button');
+        I.waitForText('Are you sure you want to remove these subscriptions?');
+        I.click('#bulk-unsubscribe-choice');
+        I.click('Continue');
+        I.waitForText('Subscription(s) removed');
```
Comment (KianKwa) [inline src/test/end-to-end/tests/verified/email-subscriptions-test.ts:243]: Think we should also go back to the main email subscription screen at this stage and confirm there is no subscription left

Comment (ashwini-mv) [reply src/test/end-to-end/tests/verified/email-subscriptions-test.ts:242]: Fixed :)

File: src/test/end-to-end/tests/verified/email-subscriptions-test.ts [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+    I.dontSeeCheckboxIsChecked('#CIVIL_AND_FAMILY_DAILY_CAUSE_LIST');
+    I.click('Email subscriptions');
+    I.click(locate('//tr').withText(locationName).find('a').withText('Unsubscribe'));
+    I.waitForText('Are you sure you want to remove this subscription?');
+    I.click('#unsubscribe-confirm');
+    I.click('Continue');
+    I.waitForText('Your subscription has been removed.');
+    I.deleteLocation(locationId);
+    await removeTestLocationFile(locationFileName);
+});
```
[the remark this answers is not in the document]
Comment (ashwini-mv) [reply src/test/end-to-end/tests/verified/email-subscriptions-test.ts:291]: Removed old tests

File: src/test/e2e/tests/specs.test.ts [hunk 3]
```diff
@@ -4,24 +4,17 @@ import { AdminDashboardPage } from '../PageObjects/AdminDashboard.page';
 import { SystemAdminDashboardPage } from '../PageObjects/SystemAdminDashboard.page';
 import { AlphabeticalSearchPage } from '../PageObjects/AlphabeticalSearch.page';
 import { CaseEventGlossaryPage } from '../PageObjects/CaseEventGlossary.page';
-import { CaseNameSearchPage } from '../PageObjects/CaseNameSearch.page';
```
Comment (ChrisS1512) [inline src/test/e2e/tests/specs.test.ts:7]: These files for these pages as well can also be removed too :)

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: src/test/e2e/tests/specs.test.ts (hunk 3 above)
Comment (ashwini-mv) [reply src/test/e2e/tests/specs.test.ts:7]: I will remove them :)

**Review by KianKwa — APPROVED** [review]

**Pull request merged**